library/manageiq_provider.py

```diff
@@ -109,6 +109,11 @@
       - the port used for hawkular metrics
     required: false
     default: null
+  skip_refresh_and_validation:
```

Comment:
regarding the flag name:
a. validation comes chronologically prior to refresh, so we should probably switch the order unless
b. we split it into 2 flags, so that the refresh depends on the validation (can't happen without it) and can also be skipped
c. if we decide to go with one flag, find a shorter name

@cben please also advise